packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -203,7 +204,6 @@ class Textarea extends React.Component {
         onChange={this.onChange}
         required={required}
         rows={textareaRows}
-        {...additionalTextareaProps}
```

Comment:
I think this is *technically* non-passive, but needed to be reorder to work appropriately

Reply:
I wouldn't worry too much about the passivity here IMO. Yeah, it technically is non passive, but I can't envision we have a lot of use cases for this scenario or needing to override classes, rows, and the other first class attributes.